resources/assets/sass/app.scss

```diff
@@ -10,8 +10,21 @@ $plyr-control-color: $colorHighlight;
 $plyr-control-bg-hover: $colorHighlight;
 $plyr-volume-track-height: 6px;
 $plyr-control-spacing: 6px;
+
 @import "vendor/_plyr.scss";
 
+.plyr {
```

Comment:
I haven't tested the PR yet, but can't we move this to `index/footer.vue` instead?